Packs/GroupIB_ThreatIntelligenceAttribution/ReleaseNotes/1_2_0.md

```diff
@@ -0,0 +1,82 @@
+#### Classifiers
+##### Group-IB Threat Intelligence & Attribution (classifier)
+- Added condition for GIB Brand Protection Domain.
+
+#### Incident Fields
+- **GIB Phishing Domain**
+- **GIB Title**
+- **GIB Address**
+- **GIB Date Expired**
+- **GIB ID**
+- **GIB Email**
+- **GIB Date Created**
+- **GIB Phishing Type**
+- **GIB Person**
+- **GIB HTML**
+- **GIB Favicon**
+- **GIB Name Servers**
+- **GIB Phishing Status**
+- **GIB Screenshot**
+- **GIB Related Indicators Data**
+
+#### Incident Types
+- **GIB Brand Protection Phishing Kit**
+- **GIB Brand Protection Domain**
+
+#### Integrations
+##### Group-IB Threat Intelligence & Attribution Feed
+- Code was adjusted to work similar to Incident Integration.
+##### Group-IB Threat Intelligence & Attribution
+- Fixed issue with loops in compromised/breached collection.
+- Added processing for new bp/domain collection.
+
+#### Layouts
+##### GIB Brand Protection Phishing Kit Layout
+- Got better UI.
+
+##### GIB Compromised IMEI Layout
+- Got better UI.
+
+##### GIB Compromised Mule Layout
+- Got better UI.
+
+##### GIB OSI Git Leak Layout
+- Got better UI.
+
+##### GIB OSI Public Leak Layout
+- Got better UI.
+
+##### GIB Targeted Malware Layout
+- Got better UI.
+
+##### GIB Victim IP Layout
+- Got better UI.
+
+##### GIB Brand Protection Phishing Layout
+- Supplemented with the new fields.
+
+##### GIB Compromised Account Layout
+- Supplemented with the new fields.
+
+##### GIB Compromised Card Layout
+- Supplemented with the new fields.
+
+##### GIB Data Breach Layout
+- Supplemented with the new fields.
+
+##### New: GIB Brand Protection Domain Layout
+- Added layout for new Incident Type.
+
+#### Mappers
+##### Group-IB Threat Intelligence & Attribution (mapper)
+- All new fields were mapped and some of the old fields got new routes.
+
+#### Playbooks
+##### Incident Postprocessing - Group-IB Threat Intelligence & Attribution
+- Changed if/else route for visual clarity.
```

Comment:
@EvgeniyMeteliza So you decided that you don't want to add a task that checks if integration is available?

Reply:
@evisochek Yeah, we discussed this with Darya, and she said that it will lead to breaking backward compatibility, and I will need to create v2 of the playbook which I want to avoid. At least before I collect a somewhat significant amount of changes.